.github/workflows/coverage.yml

```diff
@@ -35,14 +35,26 @@ jobs:
         with:
           toolchain: ${{ matrix.version }}-x86_64-unknown-linux-gnu
           profile: minimal
-          components: llvm-tools-preview 
+          components: llvm-tools-preview
           override: true
 
+      - name: Install virtualenv
+        run: pip install virtualenv
+
+      - name: Create virtualenv
+        run: virtualenv venv 
+
+      - name: Install dependencies
+        run: make env
+
       - name: generate coverage
         run: make coverage
-
+        env:
+          DYNACONF_GITEA__HOST : "https://git.batsense.net"
```

Comment:
Wouldn't it be a better approach to have the GITEA_HOST variable present in the dynaconf settings as well?

Reply:
I didn't want to hard code it in the settings but if the user wants to store the configuration a file, provisions are available for them to use [settings.toml](https://github.com/forgefedv2/interface/blob/master/config/settings.toml) and [.env](https://github.com/forgefedv2/interface/blob/master/config/.env-sample) files